### PR TITLE
mock-array: use clock tree insertion latency of macros to reduce skew

### PR DESCRIPTION
**Pull request by oharboe**
From 400 to 250ps clock period.

There is a tiny negative slack, not a bad thing in testing, but perhaps that will go away once CTS improvements are done.

### CODE REVIEW

**Comment by oharboe** [comment]
@precisionmoon FYI

File: flow/designs/asap7/mock-array/Element/config.mk [hunk 1]
```diff
@@ -43,3 +43,5 @@ export PLACE_PINS_ARGS = -annealing
 
 export GND_NETS_VOLTAGES      =
 export PWR_NETS_VOLTAGES      =
+
+export CTS_ARGS = -insertion_delay -sink_clustering_enable -balance_levels -distance_between_buffers 60
```
Comment (maliberty) [inline flow/designs/asap7/mock-array/Element/config.mk:47]: Why are you adding -distance_between_buffers 60 which seems unrelated.

Comment (oharboe) [reply flow/designs/asap7/mock-array/Element/config.mk:47]: See implementation of CTS_ARGS. This is copy, paste & modify of the default CTS args.

Comment (maliberty) [reply flow/designs/asap7/mock-array/Element/config.mk:47]: That is not in the default args:

```
if {[info exist ::env(CTS_BUF_DISTANCE)]} {
  lappend cts_args -distance_between_buffers "$::env(CTS_BUF_DISTANCE)"
}
```

Comment (oharboe) [reply flow/designs/asap7/mock-array/Element/config.mk:47]: I copied and pasted from the logs, so, I guess it is...

Comment (maliberty) [reply flow/designs/asap7/mock-array/Element/config.mk:47]: Apparently it is in the asap7 platform config (which I've never noticed).

**Review by maliberty — APPROVED** [review]
